src/main/java/duke/Event.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+/**
+ * Encapsulates an Event task.
+ */
+public class Event extends Task {
+    private String time;
+
+    /**
+     * Creates an Event object.
+     * @param description The description of the Event object.
+     * @param isDone Boolean value of the status of Event.
+     * @param time The time of the Event.
+     */
+    public Event(String description, boolean isDone, String time) {
+        super(description, isDone);
+        this.time = time;
+    }
+
+    /**
+     * Returns the String representation of the Event.
+     *
+     * @return String representation of the Event.
+     */
+    @Override
```

Comment:
Good that you remembered your Override annotations! It's easy to forget it sometimes 😊